crates/sage-cli/src/input.rs

```diff
@@ -154,23 +155,38 @@ impl Input {
         let path = matches
             .get_one::<String>("parameters")
             .expect("required parameters");
-        let mut input = Input::load(path)?;
+        let mut input = Input::load(path)
+            .with_context(|| format!("Failed to read parameters from `{path}`"))?;
 
         // Handle JSON configuration overrides
         if let Some(output_directory) = matches.get_one::<String>("output_directory") {
+            log::trace!("overriding `output_directory` parameter.");
```

Comment:
Do we think this is necessary? I think that we should definitely log when we override parameters not set by the user (e.g. where `predict_rt` is turned on if `lfq` is used), but these cases were set by the user

Reply:
I can remove them. I added them to help me when working on validating arguments from the CLI. My thinking was to trace more things, especially alterations to the input. All the parameters are dumped anyway so not that useful that's sure.

Reply:
yes I agree with the case of `predict_rt`. Here in this PR I didn't want to contain myself to the few things I touched, not really taking decisions for the project by implementing tons of changes to tracing/logging etc.